docs/upgrading_to_sql_db_14.0.0.md

```diff
@@ -221,3 +221,7 @@ module "smysql" {
   ]
 }
 ```
+
+### Made `name_override` optional in the `read_replica` object
+The [`name_override`](https://github.com/terraform-google-modules/terraform-google-sql-db/blob/master/modules/postgresql/variables.tf#L232) attribute for [`read_replica`](https://github.com/terraform-google-modules/terraform-google-sql-db/blob/master/modules/postgresql/variables.tf#L228) is optional now. If passed, the name for the read replica will be set as such. Since [optional attributes](https://developer.hashicorp.com/terraform/language/expressions/type-constraints#optional-object-type-attributes)
+is a version 1.3 feature, the configuration will fail if the pinned version is < 1.3.
```

Comment:
nit: Let's call out the TF 1.3 requirement as the heading and provide rationale below regarding `name_override`